packages/contracts/contracts/TestContracts/MockFeeSharingProxy.sol

```diff
@@ -4,16 +4,17 @@ pragma solidity 0.6.11;
 
 import "../ZERO/ZEROToken.sol";
 
-interface MockIFeeSharingProxy {
+interface MockIFeeSharingCollector {
 	function transferTokens(address _token, uint96 _amount) external;
 }
 
 /// @dev Simple contract that will receive ZERO tokens issued to the SOV stakers.
 ///      see: https://github.com/DistributedCollective/Sovryn-smart-contracts/blob/b5bd57f9003ab95ab36e20859b662f6cd7a195b5/contracts/governance/FeeSharingProxy
```

Comment:
it will be renamed to FeeSharingCollector too in the protocol repo  
pls use `Sovryn protocol fee sharing collector contract`

Reply:
https://github.com/DistributedCollective/zero/pull/174/commits/00688cd15e330f453d4ffa48d0f77477b97bcae6